api/channel_test.go

```diff
@@ -62,7 +62,7 @@ func TestCreateChannel(t *testing.T) {
 		}
 	}
 
-	if _, err := Client.DoPost("/channels/create", "garbage"); err == nil {
+	if _, err := Client.DoApiPost("/channels/create", "garbage"); err == nil {
```

Comment:
I thought all of our Client calls are api calls?  I'm guessing I'll see something further down that will explain this.